webpack/set-webpack-public-path-plugin/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@microsoft/set-webpack-public-path-plugin",
+  "version": "0.0.0",
```

Comment:
It should get bumped to `~1.0.0` upon publish, correct?

---
In reply to: [130456310](https://github.com/Microsoft/web-build-tools/pull/288#discussion_r130456310) [](ancestors = 130456310)